modules/pel/peltool/README.md

```diff
@@ -0,0 +1,67 @@
+### Peltool usage
+To know more about peltool capabilities using the help command as shown below
+`Peltool.py --help`
+
+#### List PELs
+- Get list of PELs : `peltool.py -l` (By default serviceable PELs are listed)
```

Comment:
I feel first points could be combined. The same comment applicable to count and display section.

 - Get list of serviceable PELs: `peltool.py -l` or `peltool.py -ls`.